simulator-core/src/test/java/com/evbox/everon/ocpp/it/transactions/StartTransactionIdTokenFirstIt.java

```diff
@@ -0,0 +1,62 @@
+package com.evbox.everon.ocpp.it.transactions;
+
+import com.evbox.everon.ocpp.mock.StationSimulatorSetUp;
+import com.evbox.everon.ocpp.mock.ocpp.exchange.Authorize;
+import com.evbox.everon.ocpp.mock.ocpp.exchange.StatusNotification;
+import com.evbox.everon.ocpp.mock.ocpp.exchange.TransactionEvent;
+import com.evbox.everon.ocpp.simulator.station.actions.Plug;
+import org.junit.jupiter.api.Test;
+
+import static com.evbox.everon.ocpp.mock.constants.StationConstants.*;
+import static com.evbox.everon.ocpp.v20.message.common.IdToken.Type.ISO_14443;
+import static com.evbox.everon.ocpp.v20.message.station.StatusNotificationRequest.ConnectorStatus.OCCUPIED;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionData.ChargingState.CHARGING;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionData.ChargingState.EV_DETECTED;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionEventRequest.EventType.STARTED;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionEventRequest.EventType.UPDATED;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionEventRequest.TriggerReason.CABLE_PLUGGED_IN;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionEventRequest.TriggerReason.CHARGING_STATE_CHANGED;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+public class StartTransactionIdTokenFirstIt extends StationSimulatorSetUp {
+
+    @Test
+    void shouldStartChargingWithPreAuthorization() {
+        int seqNo = 0;
```

Comment:
shouldnt that be a shared constant?

Reply:
i disagree it should be a local variable, because it's purpose is kind of a for-loop counter